### PR TITLE
Fix AMD Dependency

### PR DESCRIPTION
**Pull request by sdqwerty**
Fix AMD Dependency, add handlebars and medium editor to the factory arguments

### CODE REVIEW

**Comment by j0k3r** [comment]
Thanks !
Could you please remove dist/* files and template.js too?

File: src/wrappers/start.js [hunk 1]
```diff
@@ -11,10 +11,10 @@
                     jQuery = require('jquery')(root);
```
Comment (orthes) [inline src/wrappers/start.js]: Maybe we should define other dependencies similarly like jQuery here, in case there are not already required

**Comment by orthes** [comment]
Great! We've only recently added AMD support and I've completely forgotten about dependencies there. My bad. Can you also add jquery.fileupload.js and jquery-sortable-min.js?

**Comment by sdqwerty** [comment]
Could you check it ?

**Pull request merged**